src/main/res/layout/filter.xml

```diff
@@ -30,6 +29,7 @@
     <android.support.v4.view.ViewPager
         android:id="@+id/pager"
         android:layout_width="match_parent"
-        android:layout_height="fill_parent"
-        android:layout_below="@id/sliding_tabs"/>
+        android:layout_height="match_parent"
```

Comment:
"fill parent" is deprecated in favor of "match parent". Not worth doing a pass just for this, but I'll change it whenever I'm changing an xml file anyway.